### PR TITLE
improve internationalization

### PR DESCRIPTION
**Pull request by rbicker**
As discussed in #930:
 
- I have added i18next-browser-languagedetector to automatically detect the browser's language. If the requested language json file cannot be found a fallback to the language defined as in REACT_APP_FALLBACK_LANGUAGE ("en" by default) happens. This enables users to run multilingual installations of yopass.
- I have restructured the translations to use descriptive identifiers instead of the text fragments as keys. I have structured the translations based on the files where the text fragments are being used. These changes improve the maintainability for custom translations. Basically it means that people do not need to change their translations if you do changes to the wording of an existing text fragment.
- I have made some minor adjustments in Attribution.tsx so people doing translations can get acknowledged for their work by adding "translatorName" and "translatorLink" to their language json file. I hope this is ok for you.

As these are rather fundamental changes it would be great to merge them before doing further work on the user interface in case you agree with this pull request. If you have suggestions on how to further improve the provided code, please let us know!

I am also willing to do further work, like providing instructions on how people can write their own translations, once this pull request is being merged.

Thanks!

### CODE REVIEW

File: website/src/shared/Attribution.tsx [hunk 1]
```diff
@@ -10,12 +10,25 @@ const useStyles = makeStyles((theme) => ({
...
+  };
+
   return (
     <Container className={classes.attribution}>
       <Typography variant="body2" color="textSecondary" align="center">
-        {t('Created by')}{' '}
+        {t('attribution.createdBy')}{' '}
         <Link href="https://github.com/jhaals/yopass">Johan Haals</Link>
       </Typography>
+      {t('attribution.translatorName') !== '' ? translationAttribution() : ''}
```
Comment (jhaals) [inline website/src/shared/Attribution.tsx:31]:
```suggestion
      {t('attribution.translatorName') && translationAttribution()}
```

**Comment by jhaals** [comment]
Looks great overall 👍 
One thing missing is instructions for how to run with translations.

**Comment by rbicker** [comment]
Thanks for your feedback! I have added the instructions in the website's README.

**Comment by jhaals** [comment]
@rbicker I believe this is good to go except for the failing test. Please run `yarn format` and commit the updates to README.md.

**Pull request merged**